integration-tests/structured-logging/__tests__/to-do.js

```diff
@@ -295,16 +295,19 @@ describe(`develop`, () => {
     describe(`process.kill`, () => {
       let events = []
 
-      beforeAll(async () => {
+      beforeAll(done => {
```

Comment:
Either the process can finish initial tasks (finishedPromise represents this) or it can be killed (in which case `finishedPromise` is irrelevant)

However, even if it is killed (which sends a `SIGTERM`) it is not guaranteed to finish synchronously. We should be able to add a handler on `exit` and await that but I got lazy and added a 5 second time out for now. 

Feel free to clean this up.